src/libraries/restore/runtime/runtime.depproj

```diff
@@ -69,7 +69,6 @@
       <BinPlaceItem Include="@(MonoIncludeFiles)">
         <DestinationSubDirectory>include/%(RecursiveDir)</DestinationSubDirectory>
       </BinPlaceItem>
-      <BinPlaceItem Remove="@(BinPlaceItem)" Condition="'$(RuntimeFlavor)' != 'Mono' and '%(BinPlaceItem.Filename)' == 'System.Private.CoreLib'" />
```

Comment:
Is this change intended? Why is it necessary?

Reply:
I added it back when I merged the Trimming infrastructure because of how we used to binplace the runtime packs but it was actually not required so I'm just cleaning that up now.